lib/config/configure.js

```diff
@@ -76,7 +76,9 @@ module.exports = _.curry(function(rascalConfig, next) {
 
   function configureManagementConnection(vhostConfig, vhostName, connection) {
     _.defaultsDeep(connection.management, { hostname: connection.hostname, auth: connection.auth });
-    connection.management.auth = connection.auth || connection.management.user + ':' + connection.management.password;
```

Comment:
since `_.defaultsDeep` is already done on `connection.auth` we only need to override if they are defined